### PR TITLE
feat: Added APIs for adding removing namespace maintainers

### PR DESCRIPTION
**Pull request by arteevraina**
- [x] Don't allow package maintainers to remove other maintainers. Only namespace admins should be able to do that.
- [x] Added API to add namespace maintainers.
- [x] Added API to remove namespace maintainers.
- [x] Remove duplicate conditions from the remove package maintainers API

### CODE REVIEW

**Comment by arteevraina** [comment]
cc: @minhqdao, @perazz

File: flask/user.py [hunk 1]
```diff
@@ -326,6 +319,117 @@ def remove_maintainers_from_package(username):
...
+def add_maintainers_to_namespace(username):
+    uuid = request.form.get("uuid")
+    username_to_be_added = request.form.get("username")
+    namespace = request.form.get("namespace")
+    if not uuid:
+        return jsonify({"message": "Unauthorized", "code": 401}), 401
+    
+    # Validating the data coming with request.
+    if not uuid:
+        return jsonify({"message": "Unauthorized", "code": 401}), 401
```
Comment (minhqdao) [inline flask/user.py]: Duplication

File: flask/user.py
Comment (minhqdao) [inline flask/user.py:1]: I'd say that the code looks reasonable. But how can we test it in real world? I also think that you should write tests to make sure that these things don't accidentally break in the future.

**Review by henilp105 — APPROVED** [review]
thanks @arteevraina , other than few suggestions from @minhqdao , looks good to me.

File: flask/user.py
Comment (arteevraina) [reply flask/user.py:1]: Thanks for the review @minhqdao. 
I am thinking to have a deployed url with the test data that reviewers can use for testing. 
Regarding testing, I think I & @henilp105 should explore more about this and start writing it.

**Comment by arteevraina** [comment]
I am going to merge this branch. 
As said, I have been trying to test up docker containers & test environment for the project. I will create a separate PR for that and add some `testcases` to that as well. After that we can create small PRs to test all the APIs @henilp105

**Pull request merged**